--- ksquirrel-libs-0.7.0-pre2/admin/cvs.sh.orig	2007-04-09 22:37:22.759622872 +0200
+++ ksquirrel-libs-0.7.0-pre2/admin/cvs.sh	2007-04-09 22:38:28.058304425 +0200
@@ -64,7 +64,7 @@
     echo "*** KDE requires automake 1.6.1 or newer"
     exit 1
     ;;
-  automake*1.6* | automake*1.7* | automake*1.8* | automake*1.9* ) : ;;
+  automake*1.6* | automake*1.7* | automake*1.8* | automake*1.9* | automake*1.10*) : ;;
   "" )
     echo "*** AUTOMAKE NOT FOUND!."
     echo "*** KDE requires automake 1.6.1 or newer"
